src/book.rb

```diff
@@ -0,0 +1,14 @@
+class Book
+  attr_accessor :title, :author
+  attr_reader :rentals
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
+
+  def add_rental(rental)
+    @rentals << rental
+  end
```

Comment:
- [ ] As I suggested you in the person class to implement the `add_rental` method, For this class too the book itself should be sent as a parameter to create the new rental achieving the 'has-many' association.
take a look here :
```ruby
def add_rental(person, date)
    Rental.new(date, self, person)
  end
```